sumpy/expansion/local.py

```diff
@@ -506,13 +521,13 @@ class LinearPDEConformingVolumeTaylorLocalExpansion(
 @dataclass(frozen=True)
 class FourierBesselLocalExpansionMixin(LocalExpansionBase, ABC):
 
-    m2l_translation_override: M2LTranslationBase | None = field(
-                                                kw_only=True, default=None)
+    m2l_translation_override: M2LTranslationBase | None = (
+            field(kw_only=True, default=None))
 
     @property
     @abstractmethod
     def mpole_expn_class(self) -> type[HankelBased2DMultipoleExpansion]:
-        ...
+        pass
```

Comment:
I think `...` is better for the body of abstract methods, since it's visually distinct from an empty method.